packages/kolibri-tools/lib/i18n/crowdin.py

```diff
@@ -301,32 +295,34 @@ def download_translations(branch, project, key, login, locale_data_folder):
     """
     logging.info("Crowdin: downloading '{}'...".format(branch))
 
-    # delete previous files
-    _wipe_translations(locale_data_folder)
-
     DOWNLOAD_URL = CROWDIN_API_URL.format(
         proj=project,
         key=key,
         username=login,
         cmd="download/all.zip",
-        params="&branch={branch}&language={language}",
+        params="&branch={branch}".format(branch=branch),
```

Comment:
This will download about 10 extra languages. However, this might actually be faster than doing a couple dozen queries individually so maybe a net performance win?

Reply:
Also, when I looked at the zip files being downloaded, they didn't seem to be language specific - they all contained all the languages.

Reply:
brilliant